src/table.mjs

```diff
@@ -231,3 +271,91 @@ function likeOperand(operand) {
   return {...operand, value: `%${operand.value}%`};
 }
 
+// This function applies table cell operations to an in-memory table (array of objects).
+export function __table(source, operations) {
+  let {schema, columns} = source;
+  for (const {type, operands} of operations.filter) {
+    const column = operands.find(({type}) => type === "column").value;
```

Comment:
According to our schema, the column operand must always be listed first. So, in theory we could write it faster like this:

```suggestion
    const [{value: column}] = operands;
```